src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -8,11 +8,9 @@ module.exports = {
         '_id',
         'name',
         'email', // temp, keep the email internally
-        'access_keys',
```

Comment:
@naveenpaul1 if it helps I added a test here:
https://github.com/noobaa/noobaa-core/pull/8055/files#diff-a2598e99155f4104aee08363d420621d1489dcba5085a218b120741db65c67ac

Since the anonymous account and IAM account user (without access key) have similar attributes I will also share the way I create a new user (all were not reviewed yet):
https://github.com/noobaa/noobaa-core/pull/8055/files#diff-e17fde3b7bbe324918c2a7a422770715bd439ab6de21cc76f6679a0b7f31ecd0R442-R464
(if the link doesn't work, search `_new_user_defaults`).
cc: @romayalon @guymguym